src/commands/registerCommands.ts

```diff
@@ -86,8 +90,10 @@ export function registerCommands(): void {
     registerCommandWithTreeNodeUnwrapping('appService.toggleAppSettingVisibility', async (actionContext: IActionContext, node?: AppSettingTreeItem) => { await nonNullValue(node).toggleValueVisibility(actionContext); }, 250);
     registerCommandWithTreeNodeUnwrapping('appService.ViewCommitInGitHub', viewCommitInGitHub);
     registerCommandWithTreeNodeUnwrapping('appService.ViewProperties', viewProperties);
-    registerSiteCommand('appService.Deploy', unwrapArgs(deploy));
-    registerSiteCommand('appService.DeploySlot', unwrapArgs(deploySlot));
-    registerSiteCommand('appService.Redeploy', unwrapArgs(redeployDeployment));
-    registerSiteCommand('appService.viewDeploymentLogs', unwrapArgs(viewDeploymentLogs));
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    //@ts-ignore
```

Comment:
If I recall correctly, I think that what you're ignoring here is fixed by [Alex's PR](https://github.com/microsoft/vscode-azuretools/pull/1492).

Was the ts error related to the arguments not matching the command callback or something?

Reply:
Yup Alex had recommended just ignoring the error with a ts-ignore since I guess he was working on that PR, but I can remove those once the changes are in.